todo.sh

```diff
@@ -1134,11 +1134,11 @@ case $action in
         fi
     else
         sed -i.bak \
-            -e $item"s/^\((.) \)\{0,1\} *$3 */\1/g" \
-            -e $item"s/ *$3 *\$//g" \
-            -e $item"s/  *$3 */ /g" \
-            -e $item"s/ *$3  */ /g" \
-            -e $item"s/$3//g" \
+            -e "$item""s/^\((.) \)\{0,1\} *$3 */\1/g" \
+            -e "$item""s/ *$3 *\$//g" \
+            -e "$item""s/  *$3 */ /g" \
+            -e "$item""s/ *$3  */ /g" \
+            -e "$item""s/$3//g" \
```

Comment:
I would rather combine all of these into a single quoted string:

    "${item}s/$3//g"

Of course, it works this way, too, but it looks weird.

Reply:
@inkarkat I'm all for combining them all. Let me update it.